rfc/cart-api-v1.md

```diff
@@ -0,0 +1,84 @@
+# Version 1
+
+| Issue | Status |
+| ----- | ------ |
+| #88   | Draft  |
+
+## Data Models
+
+### `Cart`
+| Property | Type | Description |
+| --- | --- | --- |
+| `id` | UUID-like `str` | Cart ID, auto-generated by the web API |
```

Comment:
Type should just be UUID. `-like` is fuzzy and `str` is an implementation detail.